test/button.html

```diff
@@ -49,23 +48,6 @@
 			button.primary = true;
 			expect(button.hasAttribute('primary')).to.be.true;
 		});
-
```

Comment:
Expected these tests to go away?

Reply:
Thanks, nope.  Looks like these weren't added to hybrid so they are get'n clobbered.